packages/acceptance-tests/pkg-tests-specs/sources/commands/workspaces/foreach.test.js

```diff
@@ -329,16 +329,65 @@ describe(`Commands`, () => {
       makeTemporaryEnv(
         {
           private: true,
-          workspaces: [`packages/*`]
+          workspaces: [`packages/*`],
         },
-        async ({ path, run }) => {
+        async ({path, run}) => {
           await setupWorkspaces(path);
 
           await run(`install`);
           await expect(run(`workspaces`, `foreach`, `--parallel`, `--jobs`, `1`, `run`, `print`)).rejects.toThrowError(/jobs must be greater/);
         }
       )
     );
+
+    test(`can run on public workspaces only`, makeTemporaryEnv(
+      {
+        private: true,
+        workspaces: [`packages/*`],
+      },
+      async ({path, run}) => {
+        await writeFile(`${path}/.yarnrc.yml`, `plugins:\n  - ${JSON.stringify(require.resolve(`@berry/monorepo/scripts/plugin-workspace-tools.js`))}\n`);
```

Comment:
This was confusing for me when using the release candidate locally as well. While `yarn version check` tells me that the `@berry/cli` depends on workspace-tools I still have to manually add. I understand that this plugin is experimental but if it's not enabled by default in the cli it shouldn't be listed as a dependent, no?

Reply:
It seems that `plugin-workspace-tools` is listed as a dependency for `berry-cli` but you're right, it probably shouldn't. My best guess atm is that it got added when we were developing this plugin, and never got removed. The `plugin-typescript` package is in the same situation.